waspc/packages/ts-inspect/test/exportTests/complex.ts

```diff
@@ -0,0 +1,15 @@
+export default function isEvenOrOdd(n: number): boolean {
```

Comment:
What about exporting constants, do you differentiate those from functions?

```ts
export const name = "Craig";
```

Reply:
Exporting a constant is counted as a named export, it is not differentiated from functions. I added a new test case to document/check this behavior.